src/Message/Communicate.cpp

```diff
@@ -52,10 +52,10 @@ Communicate::~Communicate()
 //exclusive:  MPI or Serial
 #ifdef HAVE_MPI
 
-Communicate::Communicate(const mpi3::communicator& in_comm) : d_groupid(0), d_ngroups(1), GroupLeaderComm(nullptr)
```

Comment:
Is this just a temporal workaround?

Reply:
`mpi3::communicator const&` cannot perform the operations desired at this point. A non-const version can.